File: FullConcentration3/app/src/main/java/org/texchtown/fullconcentration3/LockActivity.java
package org.texchtown.fullconcentration3;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.app.KeyguardManager;
import android.os.Build;
import android.os.Bundle;
import android.view.WindowManager;

public class LockActivity extends AppCompatActivity {

    //widget
    Toolbar toolbar_lock;
    ActionBar actionBar_lock;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lock);

        //set toolbar as actionbar
        toolbar_lock = findViewById(R.id.toolbar_lock);
        setSupportActionBar(toolbar_lock);
        actionBar_lock = getSupportActionBar();

        actionBar_lock.setDisplayHomeAsUpEnabled(true);


        //allowing activity ui over the lock screen
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O_MR1) {
            setShowWhenLocked(true);
            setTurnScreenOn(true);
            KeyguardManager keyguardManager = (KeyguardManager) getSystemService(this.KEYGUARD_SERVICE);
            keyguardManager.requestDismissKeyguard(this, null);
        } else {

            getWindow().addFlags(WindowManager.LayoutParams.FLAG_DISMISS_KEYGUARD |
                    WindowManager.LayoutParams.FLAG_SHOW_WHEN_LOCKED |
                    WindowManager.LayoutParams.FLAG_TURN_SCREEN_ON |
                    WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
        }

    }
}

File: IBK_bank_home/app/src/main/java/org/texchtown/ibk_bank_home/LockActivity.kt
package org.texchtown.ibk_bank_home

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import org.texchtown.ibk_bank_home.databinding.ActivityAnimationBinding
import org.texchtown.ibk_bank_home.databinding.ActivityLockBinding

class LockActivity : AppCompatActivity() {

    private lateinit var binding: ActivityLockBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityLockBinding.inflate(layoutInflater)
        setContentView(binding.getRoot())

        binding.button.setOnClickListener() {
            val intent = Intent(this, MainActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}